static/styles/portico/portico-signin.scss

```diff
@@ -682,6 +682,15 @@ button.login-social-button {
     }
 }
 
+button#login_auth_button_apple,
+button#register_auth_button_apple {
```

Comment:
Let's use CSS classes rather than IDs for these.  

Also, please add a comment explaining why this requires different geometry from our default social buttons.

Reply:
The IDs were generated like this `<button id="register_{{ backend.button_id_suffix }}" class="login-social-button full-width"`. Using IDs lets us require fewer changes in the HTML pages. Is it okay to add a conditional for that to have a class for this particular one?

(Added a comment. Also changed the `height: 46px` to 50px to make those have the same height as other buttons.)

Reply:
Yeah, I guess this is OK.